Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/RadioButton.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
```

Comment:
Do we need any of these platform specifics? I think these really only existed for Button so that we could change how these worked in a non breaking way. With RadioButton we can just use the defaults